test/libsolidity/smtCheckerTests/loops/while_bmc_iterations_semantics_1.sol

```diff
@@ -0,0 +1,20 @@
+contract C {
+	function f() public pure {
+		uint x = 0;
+		int y = 0;
+		while (x < 3 || y == 1) {
+			if (x >= 3)
+				y = 1;
+			++x;
+		}
+        // BMC loop iteration setting is not enough to leave the loop
+		assert(x == 3);
+		assert(y == 1);
+	}
+}
+// ====
+// SMTEngine: bmc
+// SMTSolvers: z3
+// BMCLoopIterations: 3
+// ----
+// Info 6002: BMC: 3 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

Comment:
What changed in this test?

Reply:
Nothing's changed. I didn't edit this file in the last commit. Or am I missing something? Could you clarify? @leonardoalt

Reply:
github showed this file as modified from my last review, that's why I wondered, but didn't see any changes